components/nexus-launcher/src/main/java/org/sonatype/nexus/bundle/launcher/support/DefaultNexusBundle.java

```diff
@@ -217,6 +229,19 @@ private static void sendStopToKeepAlive(final int commandPort) {
     }
   }
 
+  @Override
+  protected String composeApplicationURL() {
+    final String contextPath =
+        getConfiguration().getContextPath() == null ? "/" + getName() : getConfiguration().getContextPath();
```

Comment:
This code appears twice, could move it into a method `getContextPath` to limit duplication/inconsistencies.